tests/behaviour/contracts/largeSizeArray/large_size_array.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.0;
+
+contract WARP {
+    uint256[100000000] public data;
```

Comment:
Warp crashed with one billion, this is 100 millions (if I didn't miss any zeroes). It would be cool to have the biggest possible number here. The closest to 2**256, the best.